string-manipulation-2/src/ransom-case.js

```diff
@@ -0,0 +1,28 @@
+/* exported ransomCase */
+function ransomCase(string) {
+  var newString = [];
+  for (var i = 0; i < string.length; i++) {
+    if (i % 2 === 0) {
+      var up = string[i].toLowerCase();
+      newString.push(up);
+    } else {
+      var lo = string[i].toUpperCase();
+      newString.push(lo);
+    }
+  }
+  var ns = newString.join('');
```

Comment:
Why not just make use of concatenation rather than using an array and converting it to a string?